pinc/Project.inc

```diff
@@ -88,6 +88,38 @@ class ProjectPageException extends ProjectException
     }
 }
 
+class ProjectNotInRoundException extends ProjectException
+{
+    public function __construct($message, $code = 110)
```

Comment:
Is there a reason the specific Exceptions like `ProjectNotInRoundException` require a message argument to their constructor, rather than the constructor containing the message ("The project is not in a round") and passing it to the parent constructor. Alternatively, if there is a need for different messages for certain circumstances, could the message argument have "The project is not in a round" as a default? If this type of exception is used in multiple places, it seems as though the default message could end up appearing in the code several times.

Reply:
Interesting point; should we provide messages where it seems there could be no dependence on the context? or keep a consistent pattern and always provide default messages or not? I defer to @cpeel 's opinion.